### PR TITLE
Feature/continuous integration

### PR DESCRIPTION
**Pull request by dcuestam**
Make sure you have checked _all_ steps below.

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Benchmarks
- [x] My PR adds the following micro benchmarks __OR__ does not need benchmarks for this extremely good reason:

### Commits
- [x] My commits have been squashed if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

## License
- [x] Add a [Mozilla Public License 2.0 license header](http://mozilla.org/MPL/2.0/) to all the new files.

### Documentation
- [x] In case of new functionality, my PR adds documentation that describes how to use it.

### CODE REVIEW

File: src/main/java/io/shapelets/khiva/Statistics.java [hunk 1]
```diff
@@ -84,11 +84,11 @@ public static Array kurtosis(Array tss) {
      * Q &gt; \chi_{1-\alpha,h}^2
      * \]
      * <p>
-     * where \( \chi_{1-\alpha,h}^2 \) is the \(\alpha\)-quantile of the chi-squared distribution with ''h'' degrees of
+     * where \( \chi_{1-\alpha,h}^2 \) is the \(\alpha\)-quantile of the ch--squared distribution with ''h'' degrees of
```
Comment (otorreno) [inline src/main/java/io/shapelets/khiva/Statistics.java]: replace - with i

File: src/test/java/io/shapelets/khiva/ArrayTest.java [hunk 2]
```diff
@@ -10,11 +10,18 @@
...
+import org.junit.Before;
 import org.junit.Test;
 
 public class ArrayTest {
     private static final double DELTA = 1e-6;
 
+
+    @Before
+    public void setUp(){
+        Array.setKhivaBackend(Array.Backend.KHIVA_BACKEND_CPU);
```
Comment (otorreno) [inline src/test/java/io/shapelets/khiva/ArrayTest.java]: check if with opencl works

File: README.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+[![License: MPL 2.0](https://img.shields.io/badge/License-MPL%202.0-brightgreen.svg)](https://github.com/shapelets/khiva-python/blob/master/LICENSE.txt)  
+Build Master Branch [![Build Status](https://travis-ci.com/shapelets/khiva-java.svg?branch=master)](https://travis-ci.com/shapelets/khiva-java)
+Code Coverance Master Branch: [![codecov.io](https://codecov.io/github/shapelets/khiva-java/coverage.svg?branch=master)](https://codecov.io/github/shapelets/khiva-java)
```
Comment (avilchess) [inline README.md]: Coverage?

File: README.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
...
+Build Develop Branch [![Build Status](https://travis-ci.com/shapelets/khiva-java.svg?branch=develop)](https://travis-ci.com/shapelets/khiva-java)
+Code Coverance Develop Branch [![codecov.io](https://codecov.io/github/shapelets/khiva-java/coverage.svg?branch=develop)](https://codecov.io/github/shapelets/khiva-java)
```
Comment (avilchess) [inline README.md]: Coverage?

File: README.md [hunk 5]
```diff
@@ -1,3 +1,10 @@
...
+
+
 # README #
 This is the Khiva binding for connecting the Java programming language and the Khiva library.
```
Comment (avilchess) [inline README.md]: This is the Khiva binding for Java, it allows the usage of Khiva library from Java.

**Pull request merged**